SumDU/Resources/GoogleService-Info.plist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CLIENT_ID</key>
+	<string>9711528499-abma08bk57i1h4nbjlbjsal161ab9uoi.apps.googleusercontent.com</string>
+	<key>REVERSED_CLIENT_ID</key>
+	<string>com.googleusercontent.apps.9711528499-abma08bk57i1h4nbjlbjsal161ab9uoi</string>
+	<key>API_KEY</key>
+	<string>AIzaSyCQsDSaP_deJeZsGGXOgGlW3aKGTWSDU_o</string>
+	<key>GCM_SENDER_ID</key>
+	<string>9711528499</string>
+	<key>PLIST_VERSION</key>
+	<string>1</string>
+	<key>BUNDLE_ID</key>
+	<string>Maksym.Skliarov.SumDU</string>
+	<key>PROJECT_ID</key>
+	<string>sumdu-4363a</string>
+	<key>STORAGE_BUCKET</key>
+	<string>sumdu-4363a.appspot.com</string>
+	<key>IS_ADS_ENABLED</key>
+	<false></false>
```

Comment:
@Gomer0513 it's a very weird format, have you just dragged and dropped this file into the project?

Reply:
@skliarov year. I've just added it to the project.